platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/framework/internal/MeasuredBlockingQueue.java

```diff
@@ -33,8 +33,8 @@
  */
 class MeasuredBlockingQueue<T> extends AbstractBlockingQueue<T> {
 
-    static final String QUEUE_MAX_SIZE_SUFFIX = "-queueMaxSize";
-    static final String QUEUE_MIN_SIZE_SUFFIX = "-queueMinSize";
+    static final String QUEUE_MAX_SIZE_SUFFIX = "_queueMaxSize";
+    static final String QUEUE_MIN_SIZE_SUFFIX = "_queueMinSize";
```

Comment:
Will this change metrics names that are current used?

Reply:
It will not change the name on the Prometheus side; they were already called this after the adapter fixed the name.